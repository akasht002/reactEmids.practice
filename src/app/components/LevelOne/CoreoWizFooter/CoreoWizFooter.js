import React from "react";
import {Button} from "../../";

class CoreoWizFooter extends React.Component {

    render() {
        return (
            <div className="container-fluid bottom-footer-content p-3 boxShadowTop">
                <div className="row">
                    <div className="col-md-12">
                        <Button
                            type="button"
                            classname="btn outline btn-outline-primary mx-2 float-left"
                            label="Cancel"
                            onClick={this.props.onCancelClick}
                            disable={false}
                        />
                        {this.props.displayNextButton &&
                        <Button
                            type="button"
                            classname="btn outline btn-primary mx-2 float-right"
                            label="Next"
                            onClick={this.props.onNextClick}
                            disable={this.props.isNextDisabled}
                        />}
                        {this.props.displaySubmitButton &&
                        <Button
                            type="button"
                            classname="btn outline btn-primary mx-2 float-right"
                            label="Submit"
                            onClick={this.props.onSubmitClick}
                            disable={this.props.isSubmitDisabled}
                        />}
                        {this.props.diplayPrevButton &&
                            <Button
                                type="button"
                                classname="btn outline btn-outline-primary mx-2 float-right"
                                label="Previous"
                                onClick={this.props.onPreviousClick}
                                disable={false}
                            />
                        }
                    </div>
                </div>
            </div>
        )
    }
};

export default CoreoWizFooter;