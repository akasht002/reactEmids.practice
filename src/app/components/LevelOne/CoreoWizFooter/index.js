import CoreoWizFooter from './CoreoWizFooter';

export {CoreoWizFooter};