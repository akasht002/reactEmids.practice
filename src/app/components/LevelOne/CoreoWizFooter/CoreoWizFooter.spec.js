import React from 'react';
import CoreoWizFooter from './CoreoWizFooter';
import renderer from 'react-test-renderer';

test('CoreoWizFooter Component testing', () => {
    const component = renderer.create(
      <CoreoWizFooter></CoreoWizFooter>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });