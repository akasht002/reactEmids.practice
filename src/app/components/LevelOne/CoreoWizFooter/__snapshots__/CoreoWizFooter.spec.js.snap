// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CoreoWizFooter Component testing 1`] = `
<div
  className="container-fluid bottom-footer-content p-3 boxShadowTop"
>
  <div
    className="row"
  >
    <div
      className="col-md-12"
    >
      <button
        className="btn outline btn-outline-primary mx-2 float-left"
        disabled={false}
        type="button"
      >
        Cancel
      </button>
    </div>
  </div>
</div>
`;
